sophia-vote-contract.md

```diff
@@ -61,19 +61,25 @@ We need to do this because in Sophia ML we don not have a default value of 0x0/f
 
 Next we create the vote function which looks basically like this:
 `````
-public stateful function vote(candidate: address) : bool =       is_candidate(candidate)       let new_votes_state = Call.caller :: state.votes[candidate].voters       put(state{          votes[candidate].voters = new_votes_state }) true
+stateful entrypoint vote(candidate: address) : bool =
+   is_candidate(candidate)
+   let new_votes_state = Call.caller :: state.votes[candidate].voters
+   put(state{
+      votes[candidate].voters = new_votes_state })
+   true
```

Comment:
guess there is no need to return true

Reply:
Thanks for the feedback. I have removed it. Kindly, resolve the conversation if you are satisfied with the new changes.

Reply:
still seems to be there